pkg/ebpf/encoding_functions.go

```diff
@@ -59,16 +62,43 @@ func encodeMemOpcode(op *pb.MemOpcode) (uint8, error) {
 }
 
 // EncodeInstructions transforms the given array to ebpf bytecode.
```

Comment:
update the comment to reflect that you are also encoding func info

Reply:
Done